### PR TITLE
Create todo list

### PR DESCRIPTION
**Pull request by chudisoft**
Setup Webpack.
Create to-do list form and list section.
Use a Javascript object to store each item entered.
Style to resemble the provided sample.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+input[type='checkbox'] {
+  scale: 1.4;
+  cursor: pointer;
+}
+
+.todo {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```
Comment (Eternalgratis) [inline src/style.css:100]: - Currently, the checkbox is almost out of the border.  Kindly add the necessary  styling to it for a better look
- Please also consider adding more styling to the input box for a  better user experience. Kindly see the highlighted part below for clarity

image

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
## Hi @chudisoft , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there

## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ Linter error checks validated

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (chudisoft) [reply src/style.css:100]: image
Attached is my design.
Am not sure why you got that error, but it looks as if the page is not properly loaded on your end.

**Review by sumairq — APPROVED** [review]
### Hi @chudisoft , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**